File: backend/models/player.py
from enums.roles import Role
from enums.fundamentals import FundamentalType

class Player:
    def __init__(self, id, name, surname, number, role: Role):
        self.id = id
        self.name = name
        self.surname = surname
        self.number = number
        self.role = role
        self.fundamentals = [FundamentalType]

File: MatchVision/src/app/Models/Player.ts
export enum Role {
  SETTER = 'SETTER',
  OUTSIDE_HITTER = 'OUTSIDE_HITTER',
  MIDDLE_BLOCKER = 'MIDDLE_BLOCKER',
  OPPOSITE_HITTER = 'OPPOSITE_HITTER',
  LIBERO = 'LIBERO'
}

export class Player {
  constructor(
    public id: number,
    public name: string,
    public surname: string,
    public number: number | null,
    public role: Role | ''
  ) {}
}